packages/loader/container-loader/src/container.ts

```diff
@@ -1192,20 +1193,23 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         } else {
             //
             // THIS IS LEGACY PATH
+            // The code is maintained for the snapshot back compat tests
             //
+            assert(createIfNotExisting === true, "Snapshot should already exist");
```

Comment:
Trying to figure out why this hasn't been shortcoded by the policy-check tool...